ray-on-gke/tpu/kuberay-tpu-webhook/Makefile

```diff
@@ -16,7 +16,19 @@ webhook:
   
 # Run against the configured Kubernetes cluster in ~/.kube/config  
 run: webhook  
-	go run ./main.go  
+	go run ./main.go
+
+# Run go fmt against code.
+fmt:
+	go fmt ./...
+
+# Run go vet against code.
+vet:
+	go vet ./...
+
+# Build and run go test against code.
+test: webhook
```

Comment:
nit: I don't think you need to run the `webhook` target before running `go test`

Reply:
[746f898](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/746f898bc47e3dd361dbde136b7a9fe92705b9aa)